wg-meetings/2016-07-14.md

```diff
@@ -0,0 +1,97 @@
+# Attendees
+* Michael Dawson (@mhdawson)
+* Arunesh Chandra (@aruneshc) - observer
+* Ian Halliday (@ianwjhalliday)
+* Sampson Gao (@sampsongao)
+* Ali Sheikh (@ofrobots)
+* Barrett Harber (@thelogicbox)
+* Patrick Mueller (@pmuellr) - lurking
+
+# Agenda
+* Stand up
+* Actions from last meeting
+* Update on work for ABI stable module API by -
+  @ianwjhalliday, @sampsongao and @mhdawson
+* milestone planning
+* Keep working in Ian’s fork or somewhere else ?
+* open issue review
+
+# Meeting minutes
+
+## Stand up
+* Michael Dawson
+  * working on getting all NaN examples running
+  * refactored to remove C++ elements from API
+  * starting to look at sqlite3 as another more significant module to port
+* Ian Halliday
+  * started getting NaN examples converted to API
+  * converted leveldown module to API, no v8 imports!
+  * will investigate performance impact to leveldown next
+* Sampson Gao
+  * Modify the init method for addons by NODE_MODULE_ABI
+  * building new tests for api/abi
+* Ali Ijaz Sheikh (@ofrobots)
+  * Steady progress on FFI proof-of-concept.
+
+## Actions from last meeting
+Michael to review README.md to see if we can improve positioning versus Nan wg.  Rolled over as he has not gotten to this yet.
+
+## Upate on ABI stable working
+
+* walked through node_jsvmapi.h to show current methods
+* walked through selection of NaN examples with comparison between
+  ABI stable and NaN/native versions.
+* Walked through level down as a larger module that has been ported observer
+* Discussed some next steps which include:
+  * Updating EPS to reflect current methods
+  * Opening issue (Michael) to discuss if we should continue to use Ian's
+  * repo or move code somewhere else.
+  * porting of sqlite3
+  * performance testing on leveldown
+
+
+Code is currently available here:
+https://github.com/ianwjhalliday/node/tree/api-opaque-prototype/src
+
+  * node_jsvmapi.cc
+  * node_jsvmapi.h
+  * node_jsvmapi_internal.h
+  * node_jsvmapi_types.h
+
+and NaN examples here:
+
+https://github.com/ianwjhalliday/node-addon-examples/tree/api-opaque-prototype
```

Comment:
we should probably also include a link to the leveldown fork

https://github.com/ianwjhalliday/leveldown/tree/api-opaque-prototype